### PR TITLE
Build stages

### PR DESCRIPTION
**Pull request by mimmi20**
I would like to use the travis build stages.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,53 +1,58 @@
 language: php
 sudo: false
+dist: trusty
```
Comment (asgrim) [inline .travis.yml]: This is implied anyway isn't it now? I thought Travis had already migrated default to `trusty`?

File: .travis.yml [hunk 2]
```diff
@@ -1,53 +1,58 @@
...
 
 php:
   - 7.0
```
Comment (asgrim) [inline .travis.yml]: Don't think we need to test on 7.0 any more. Let's bump min to 7.1.

File: .travis.yml (hunk 1 above)
Comment (mimmi20) [inline .travis.yml]: According to this post (https://blog.travis-ci.com/2017-07-11-trusty-as-default-linux-is-coming?utm_source=web&utm_medium=banner&&utm_campaign=trusty-default) the migration shall be finished at 1st of september.

File: .travis.yml (hunk 2 above)
Comment (mimmi20) [inline .travis.yml]: removed

File: .travis.yml (hunk 1 above)
Comment (mimmi20) [inline .travis.yml]: removed

**Comment by mimmi20** [comment]
@asgrim Could you review again?

**Comment by asgrim** [comment]
Thanks!

**Pull request merged**